### DISCUSSION TITLE
Bug or feature? No way to create Pull Request using github website using Windows Vista?

### DISCUSSION THREAD

**Discussion by rusefi** [community/community#22345] [New to GitHub]
<p>Short verssion: looks like no way to create pull request using github website using Windows Vista?</p>
<p>Longer version: Yes, I know IE, Opera, Chrome and Firefox current versions do not support Vista.</p>
<p>The specific problem is two-fold: “Checking mergeability… Don’t worry, you can still create the pull request.” stays forever, and “Create Pull Request” button does nothing which is a problem.</p>
<p>I’ve tried to find _any_ browser which would support Vista. So far I was able to find Lunascape6 - I know, a weird one - which allows to use WebKit rendering engine. GitHub does not show an “unsupported” version message yet still - same issue “Checking mergeability…” hanging and same “Create Pull Request” button does nothing.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/1036e8e8f246843e2c412054723a3200b5d0a65f.png" data-download-href="/uploads/short-url/2jrhyMbZ8UgajrUzCuvHhowCCGb.png?dl=1" title="no_vista.png.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7377949/181070270-e1e3efca-197e-45f5-b69c-6a8c9a5d1638.png" alt="no_vista.png.png" data-base62-sha1="2jrhyMbZ8UgajrUzCuvHhowCCGb" width="690" height="474" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/1036e8e8f246843e2c412054723a3200b5d0a65f_2_690x474.png, https://aws1.discourse-cdn.com/github/original/1X/1036e8e8f246843e2c412054723a3200b5d0a65f.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/1036e8e8f246843e2c412054723a3200b5d0a65f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/1036e8e8f246843e2c412054723a3200b5d0a65f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">no_vista.png.png</span><span class="informations">960×660 66.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/rusefi">@rusefi</a>,</p>
<p>Unfortunately, to keep up to date with current security protocols, <a href="https://help.github.com/articles/supported-browsers/" rel="nofollow noopener">GitHub is only guaranteed to work on the most current browsers</a> which means that you might be out of luck when it comes to finding a browser that works with Vista specifically.</p>
<p>Sorry that I couldn’t give a more heartening answer.</p>
